### ISSUE TITLE
installation commands using the template fail

### ISSUE THREAD

**Issue by elrayle** [projectblacklight/spotlight#2744]
### Systems

* ruby 2.6.7
* rails 6.1.3.1

### Description

Running templates from README [Installation](https://github.com/projectblacklight/spotlight#installation) instructions fail.  I tried both creating a new app from scratch using the template command for new apps.  And I tried creating the Rails app first and then running the template command for existing apps.  Both approaches fail.

### Failure

It successfully adds the blacklight and blacklight-spotlight gems.  It successfully runs `bundle install`.  It fails while running `generate 'blacklight:install'`.

```
    generate  blacklight:install
       rails  generate blacklight:install --devise
Could not find gem 'solr_wrapper (>= 0.3)' in any of the gem sources listed in your Gemfile.
Run `bundle install` to install missing gems.
```

It gets the same error running both commands.

**Comment by elrayle** [comment]
FYI... I followed the [Installation](https://github.com/projectblacklight/blacklight#installation) instructions to create a Blacklight app and the command `rails generate blacklight:install` completed without error.

**Comment by elrayle** [comment]
I tried adding `gem 'solr_wrapper', '>= 0.3'` before the `bundle install` in the template.  With that, it gets further in the `blacklight:install`, but still ultimately fails.  It complains that directory `solr` and its contents and `.solr_wrapper.yml` already exist.  But seems to happily continue.

While installing `blacklight:assets`, it finds more files that exist.  And ultimately fails...

```
    generate  blacklight:assets
       rails  generate blacklight:assets 
Running via Spring preloader in process 35666
     gemfile  bootstrap (~> 4.0)
     gemfile  twitter-typeahead-rails (0.11.1.pre.corejavascript)
   identical  app/assets/javascripts/application.js
File unchanged! The supplied flag value not found!  app/assets/config/manifest.js
       exist  app/assets/images
   identical  app/assets/stylesheets/blacklight.scss
/Users/elr37/.rvm/gems/ruby-2.6.7/gems/blacklight-7.16.0/lib/generators/blacklight/assets_generator.rb:75:in `read': No such file or directory @ rb_sysopen - app/assets/javascripts/application.js (Errno::ENOENT)
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/blacklight-7.16.0/lib/generators/blacklight/assets_generator.rb:75:in `has_blacklight_assets?'
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/blacklight-7.16.0/lib/generators/blacklight/assets_generator.rb:39:in `assets'
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/thor-1.1.0/lib/thor/invocation.rb:134:in `block in invoke_all'
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/thor-1.1.0/lib/thor/invocation.rb:134:in `each'
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/thor-1.1.0/lib/thor/invocation.rb:134:in `map'
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/thor-1.1.0/lib/thor/invocation.rb:134:in `invoke_all'
	from /Users/elr37/.rvm/gems/ruby-2.6.7/gems/thor-1.1.0/lib/thor/group.rb:232:in `dispatch'
        ....
```

**Comment by camillevilla** [comment]
@elrayle Took this for a spin just now and I think the quickstart issue is in upstream Blacklight. More notes soon.

**Comment by camillevilla** [comment]
This should get resolved once we get projectblacklight/blacklight#2442 into Spotlight.

**Comment by cbeer** [comment]
🤞 , fixed by Blacklight 7.17.
